### DISCUSSION TITLE
installation instructions for installing wiki.js seem to no working anymore

### DISCUSSION THREAD

**Discussion by FriesenRudi** [requarks/wiki#7418] [Error / Bug Report]
I followed the installation instructions for installing wiki.js on a fresh installed Ubuntu 22.04 LTS and also on Ubuntu 24.04 LTS. Ubuntu was installed as a VM on a Windows Server 2019 Hypervisor. The trouble starts at 'Setup Containers' Seem to be a problem with accessrights or so. But as it is written at this moment it just does not work. Does anybody know what's wrong?

**Comment by BPowell76** [comment]
I'm using Wiki.js "bare metal" on a Ubuntu 24.04 LTS server (upgraded from 22.04 a month or two ago) and it works fine. I haven't tried running it in a container outside of some initial testing/prototyping before deploying it at my job.

It sounds like a permissions issue though.

A little unrelated (but seems relevant), but when I needed to setup my site to use SSL certs I had to modify the service to run as root/sudo. It was either that or copy the SSL certs to the root directory for the site being served by node.

**Comment by NickelplateMedia** [comment]
**Failed Setup** Ubuntu 22.04.5 on top of a PVE 9.1.4 MiniPC
I just followed the Ubuntu directions by doing my script monkey job of copying and pasting. I'm noticing that the strings use sudo until docker is installed, but then no sudo? Isn't there a step or two missing before we can run docker without sudo? Is that what we want to be doing? Should we just use root? What would you modify to make this work?

https://docs.requarks.io/install/ubuntu

-In humble regards, Nickelplate the script monkey
